components/tag-list/tag-list-item-mixin.js

```diff
@@ -65,12 +63,27 @@ export const TagListItemMixin = superclass => class extends superclass {
 		super();
 		/** @ignore */
 		this.role = 'listitem';
+		this._id = getUniqueId();
+	}
+
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+
+		const container = this.shadowRoot.querySelector('.tag-list-item-container');
+		this.addEventListener('focus', () => container.focus());
```

Comment:
By redirecting the focus from the host, the focus styles above (`:host(:focus) .tag-list-item-container`) doesn't appear to work as expected.  What's the reason for moving the focus inside the item from the host?

Reply:
The tooltip wouldn't show up on arrow key "focus" unless the focus was placed on the `.tag-list-item-container`. ([related](https://github.com/BrightspaceUI/core/pull/2298#discussion_r845422966))

The other option I can think of here that would not involve having to move the focus within the tag-list-item is the `tag-list` adds sibling tooltips to each list tag list item on slotupdate, but that doesn't seem like a great option to me vs. just having the tooltip live in the component (though the way this has to be modified for the tooltip to appear on focus is also not great).

Reply:
Cool yeah.  So looks like the CSS selector just needs to be updated.  😄